src/components/BaseHead.astro

```diff
@@ -1,17 +1,15 @@
 ---
-// Import the global.css file here so that it is included on
-// all pages through the use of the <BaseHead /> component.
-import '../styles/global.css';
+import "../styles/global.css";
 
 export interface Props {
-	title: string;
-	description: string;
-	image?: string;
+  title: string;
+  description: string;
+  image?: string;
```

Comment:
is this change to 2 spaces from your editor settings or does astro have a style guide?

Reply:
It's pretty much the formatter.